New function to transfer between (secondary) fungible stores (#682)
aptos-labs/aptos-ts-sdk: CHANGELOG.md

## Unreleased

> - Add a function to TS SDK to transfer Fungible Assets between (secondary) fungible stores. Includes an example file `transfer_secondary_store.ts` which uses a new example Move module `secondary_store.move`.

Reviewer: ```suggestion
- Add a `transferFungibleAssetBetweenStores` function to transfer Fungible Assets between any (primary or secondary) fungible stores. 
- Include an example file `transfer_secondary_store.ts` which uses a new example Move module `secondary_store.move`.
```
